### PR TITLE
Integrated Google SignIn with Firebase

### PR DESCRIPTION
**Pull request by pandilladivya**
The following is the screenshot to show the work done.

screen shot 2017-08-19 at 1 35 20 am

@sandarumk @HimanshiJain Kindly review

### CODE REVIEW

**Comment by pandilladivya** [comment]
Okay.

**Comment by pandilladivya** [comment]
Resolved conflicts! :)

**Comment by pandilladivya** [comment]
I'll share the signedAPK with you and @sandarumk. I think the problem is with SHA key.

File: app/src/main/java/vola/systers/com/android/activities/SignInActivity.java [hunk 1]
```diff
@@ -285,40 +273,27 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
...
         if (requestCode == RC_GOOGLE_SIGN_IN) {
-            Log.d(TAG, "signin");
             GoogleSignInResult result = Auth.GoogleSignInApi.getSignInResultFromIntent(data);
-            handleGoogleSignInResult(result);
+            if (result.isSuccess()) {
+                // Google Sign In was successful, authenticate with Firebase
+                GoogleSignInAccount account = result.getSignInAccount();
+                firebaseAuthWithGoogle(account);
+            } else {
+                Log.i(TAG,"failure");
```
Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/activities/SignInActivity.java]: Should we display a toast here?

Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/activities/SignInActivity.java]: Yeah I think we should. I'll add

**Pull request merged**